src/System.Numerics.Vectors/src/System.Numerics.Vectors.csproj

```diff
@@ -0,0 +1,74 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp2.0;netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <IsPackable>false</IsPackable>
+    <VersionPrefix>4.5.0</VersionPrefix>
+    <VersionPrefix Condition="'$(IsPackable)' == 'true'">4.6.0</VersionPrefix>
+    <AssemblyVersion>4.1.4.0</AssemblyVersion>
+    <!-- NO-INCREMENT: This version is frozen for .NET Standard and .NETCoreApp because the assembly ships inbox. -->
+    <AssemblyVersion Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">4.1.4.0</AssemblyVersion>
+    <AssemblyVersion Condition="'$(IsPackable)' == 'true' and $([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">4.1.5.0</AssemblyVersion>
+    <PackageValidationBaselineVersion>4.5.0</PackageValidationBaselineVersion>
+    <IsPlaceholderTargetFramework Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETCoreApp'">true</IsPlaceholderTargetFramework>
+    <EnableDefaultItems>false</EnableDefaultItems>
+  </PropertyGroup>
+
+  <!-- On .NET Core, Vector<T> is in System.Private.CoreLib -->
+  <ItemGroup Condition="'$(IsPlaceholderTargetFramework)' != 'true'">
+    <Compile Include="System\Numerics\ConstantHelper.cs">
+      <AutoGen>True</AutoGen>
+      <DesignTime>True</DesignTime>
+      <DependentUpon>ConstantHelper.tt</DependentUpon>
+    </Compile>
+    <Compile Include="System\Numerics\Register.cs">
+      <AutoGen>True</AutoGen>
+      <DesignTime>True</DesignTime>
+      <DependentUpon>Register.tt</DependentUpon>
+    </Compile>
+    <Compile Include="System\Numerics\Vector.cs">
+      <AutoGen>True</AutoGen>
+      <DesignTime>True</DesignTime>
+      <DependentUpon>Vector.tt</DependentUpon>
+    </Compile>
+    <Compile Include="System\Numerics\Vector_Operations.cs" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(IsPlaceholderTargetFramework)' != 'true'">
+    <None Include="System\Numerics\GenerationConfig.ttinclude" />
+    <Content Include="System\Numerics\ConstantHelper.tt">
+      <Generator>TextTemplatingFileGenerator</Generator>
+      <LastGenOutput>ConstantHelper.cs</LastGenOutput>
+    </Content>
+    <Content Include="System\Numerics\Register.tt">
+      <Generator>TextTemplatingFileGenerator</Generator>
+      <LastGenOutput>Register.cs</LastGenOutput>
+    </Content>
+    <Content Include="System\Numerics\Vector.tt">
+      <Generator>TextTemplatingFileGenerator</Generator>
+      <LastGenOutput>Vector.cs</LastGenOutput>
+    </Content>
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(IsPlaceholderTargetFramework)' != 'true'">
```

Comment:
Looks to me like *some* of these were also excluded on NetFx before
https://github.com/dotnet/corefx/blob/06a86ddfefa9042b22f2bcefc9e32088e077df78/src/System.Numerics.Vectors/src/System.Numerics.Vectors.csproj#L70-L81